src/murmur/Meta.cpp

```diff
@@ -695,6 +698,8 @@ bool Meta::boot(int srvnum) {
 		return false;
 	if (! ServerDB::serverExists(srvnum))
 		return false;
+	if (! ServerDB::getConf(srvnum, "autostart",Meta::mp.bAutoStart).toBool())
+		return false;
```

Comment:
I don't think it's a good idea to handle it this way. When this function is called, the server should start.
Instead I think this setting should be checked inside `Meta:bootAll()` - maybe even extend the function's signature to be `void bootAll(bool respectAutostartConfig)` and only check the config if `respectAutostartConfig` is `true`

Reply:
> I don't think it's a good idea to handle it this way. When this function is called, the server should start.

Yes, bad idea. This does actually prevent the server from starting at all when autostart=false:

./murmur-cli --insecure=true server start 2
{"error":"rpc error: code = Unknown desc = server could not be started, or is already started"}